addons/web_editor/static/src/xml/backend.xml

```diff
@@ -17,4 +17,8 @@
         </span>
     </t>
 
+    <t t-extend="wysiwyg.widgets.link">
+        <t t-jquery="div.o_new_window" t-operation="replace"></t>
+    </t>
```

Comment:
Can't you just add a `t-if` on the related section in the base template and check you `forceNewWindow` parameter? That way it is dependent on the parameter and not the backend/frontend should we even use the parameter elsewhere.